From: Dermot Bradley <dermot_bradley@yahoo.com>
Date: Fri, 29 May 2020 19:58:07 +0100

Fix the path to the audispd binary.

There is no point upstreaming this fix as the next release (3.0
in development) no longer uses this configuration setting.

--- a/init.d/auditd.conf
+++ b/init.d/auditd.conf
@@ -13,7 +13,7 @@
 num_logs = 5
 priority_boost = 4
 disp_qos = lossy
-dispatcher = /sbin/audispd
+dispatcher = /usr/sbin/audispd
 name_format = NONE
 ##name = mydomain
 max_log_file_action = ROTATE
